### ISSUE TITLE
Nvidia HDR support is unclear/misleading

### ISSUE THREAD

**Issue by 825i** [gianklug/wearewaylandnow#31]
Hi, please don't take this as a complaint but merely feedback.

First of all, thanks a LOT for improving and maintaining on the arewewaylandyet repo and creating wearewaylandnow.

Specifically I've been waiting years for a method to use HDR with Nvidia on Wayland/Linux in general. Unfortunately, it's not clear on the site, whether this is at all supported. There *is* some information for HDR but it all pertains to AMD which has been supported now for quite a while.

Specifically in the HDR section on the site, link 1 has a single comment from December 2023 about Nvidia not working. Link 2 has zero mention for the method of getting HDR to work on Nvidia at all. All the steps are again only working for those using an AMD GPU and drivers.

_tl;dr - It would make sense to clarify that HDR support only works on AMD, and add a second entry for HDR under "nvidia" that states it does not work_.

Thanks!

**Comment by gianklug** [comment]
If you have time for it, a PR would be appreciated 😄

Otherwise, I'll do it at some point over the next weeks.
